sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/event_processor.py

```diff
@@ -308,11 +322,7 @@ def start(self):
                         error,
                     )
                     if self._error_handler:
-                        self._handle_callback(
-                            self._error_handler,
-                            self._partition_contexts[partition_id],
-                            error,
-                        )
+                        self._process_error(self._partition_contexts[partition_id], error)
```

Comment:
This statement does not need to be within a `if self._error_handler` statement, because this check is already done with `def _process_error`

Reply:
you're right, another 'doubled' place :P